lib/xandra/cluster/pool.ex

```diff
@@ -340,8 +331,10 @@ defmodule Xandra.Cluster.Pool do
         %__MODULE__{} = data
       ) do
     if data.peers[peername] do
-      data = put_in(data.peers[peername].status, :down)
-      data = stop_pool(data, data.peers[peername].host)
+      data = set_host_status(data, peername, :down)
+      host = data.peers[peername].host
+      data = stop_pool(data, host)
+      data = maybe_start_pools(data)
```

Comment:
Can we leave a comment here that explains why we are starting pools after stopping this one? I know why, just for the poor soul (could be one of us) looking at all this months from now 😄